IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/Types/IfcRailwayPartTypeEnum/Documentation.md

```diff
@@ -1 +1,21 @@
 The IfcRailwayPartTypeEnum defines the range of different types of railway part that can be specified.
+
+The Spatial structure of a Railway is sub-divided into several elements for lateral and vertical decomposition:
+
+* TRACKSTRUCTURE is for the Track (vertical decomposition)
+* SUPERSTRUCTURE is for the upper part of the railway (vertical decomposition)
+* LINESIDESTRUCTURE is for the lateral part of the railway  (lateral decomposition)
```

Comment:
couldn't LINESIDE be used for vertical breakdown too? I remember your picture
![Picture1](https://user-images.githubusercontent.com/62438551/125086078-8fc57f00-e0cb-11eb-8b91-198997e399e2.png)

Reply:
@evandroAlfieri
Yes, but in the tabular shared with Infra it is presented as Lateral decomposition so I  limit the documentation to this aspect